.github/workflows/sudo-python-tests.yml

```diff
@@ -9,7 +9,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: Sudo python tests
+name: AGW Test Sudo Python
```

Comment:
Those are integration tests, right?
In that case the naming scheme from above would apply. E.g. `AGW Test Integration Sudo Python`.

Reply:
My understanding is that despite them being run in the integration test workflow in the past, they're not actually integration tests. Can you please confirm @nstng?

Reply:
They are kind of in a gray zone - unit tests that need a running ovs. IMO I would not call them integration tests because they do not need magma services - I think `AGW Test Sudo Python` is a good name.